apps/full-site-editing/full-site-editing-plugin/premium-content/subscription-service/class-jetpack-token-subscription-service.php

```diff
@@ -32,7 +32,8 @@ public function get_site_id() {
 	 * @inheritDoc
 	 */
 	function get_key() {
-		$token = Manager::get_access_token();
+		$connection = new Manager();
```

Comment:
Internally is this a singleton, or do we create a new instance on each get_key call?

Reply:
No, no singleton: https://github.com/Automattic/jetpack/blob/master/packages/connection/src/class-manager.php#L64

It does have some interesting configuration requirements for listeners, but they don't appear necessary in this case, looking at other instances: https://github.com/Automattic/jetpack/blob/master/packages/connection/src/class-client.php#L55

Reply:
This is probably safe then. Definitely a PHP wat though that it allowed it in the first place. (Usage was only recently deprecated in 7) https://www.php.net/manual/en/language.oop5.static.php